src/main/java/org/opendevstack/provision/services/BitbucketAdapter.java

```diff
@@ -477,14 +477,14 @@ public static boolean containsGroup(JsonNode json, String group) {
           repo.setAdminGroup(project.getProjectAdminGroup());
```

Comment:
This part seems now to not be covered with a unit test. Can you please extend it?

Reply:
Done